src/components/CommentList/CommentList.tsx

```diff
@@ -9,7 +9,7 @@ type Props = {
 };
 
 export const CommentList: React.FC<Props> = ({ comment }) => (
-  <div className="CommentList">
+  <ul className="CommentList" style={{ listStyleType: 'none' }}>
```

Comment:
do not add styles via `style` attribute (especially when you have a separate `css` file with styles) - add styles in CSS (scss in this case) file